xamarin/Xamarin.Forms
# [Android] Fix diagonal LinearGradientBrush issues

### Description of Change ###

Fix diagonal LinearGradientBrush issues ( platform differences with iOS).

### Issues Resolved ### 

- fixes #11571 
- fixes #11715

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Launch Core Gallery and navigate to the issue 11571 and 11715 on Android and iOS. Verify that renders exactly the same on Android as on iOS.

### PR Checklist ###
